design/authn_azure/authn_azure_solution_design.md

```diff
@@ -420,44 +420,44 @@ Azure authenticator performance should conform with our other authenticators wit
+| 7  | authn-azure is/is not whitelisted in ENV or enabled in DB (3 separate tests)                                                | VM sends authentication request to Conjur with valid Azure token    | Return 401 Unauthorized error and `AuthenticatorNotEnabled` message appears in log       | Integration               | <ul><li> [ ] Tested</li></ul>| No        |
+| 8  | Policy is loaded w/o `provider-uri` variable                                                                                | VM sends authentication request to Conjur with valid Azure token    | Return 401 Unauthorized error and `RequiredResourceMissing` message appears in log       | Integration               | <ul><li> [ ] Tested</li></ul>| Yes       |
+| 9  | Policy is loaded with `provider-uri` variable but a value is never added                                                    | VM sends authentication request to Conjur with valid Azure token    | Return 401 Unauthorized error and `RequiredSecretMissing` message appears in log         | Integration               | <ul><li> [ ] Tested</li></ul>| No        |
+| 10 | Policy is loaded w/o defining authenticator webservice                                                                      | VM sends authentication request to Conjur with valid Azure token    | Return 401 Unauthorized error and `WebserviceNotFound` message appears in log            | Integration               | <ul><li> [ ] Tested</li></ul>| No        |
+| 11 | An invalid/non-existent `provider-uri` (Conjur couldn't connect to provider)                                                | VM sends authentication request to Conjur with valid Azure token    | Return 504 GatewayTimeout error and `ProviderDiscoveryTimeout` message appears in log    | Integration               | <ul><li> [ ] Tested</li></ul>| No        |
+| 12 | Conjur can make contact with provider but couldn't confirm token signature                                                  | VM sends authentication request to Conjur with invalid Azure token  | Return 502 Bad Gateway error and `ProviderTokenInvalid` message appears in log           | Integration               | <ul><li> [ ] Tested</li></ul>| Yes       |
+| 13 | One or more annotations combination is incorrect or missing from the host policy (6 separate tests, see below explanation)  | VM sends authentication request to Conjur with valid Azure token    | Return 401 Unauthorized error and `RoleMissingAnnotations` message appears in log        | Integration               | <ul><li> [ ] Tested</li></ul>| partially |
+| 14 | Expected field from request are missing (missing "jwt" in request)                                                          | VM sends authentication request to Conjur with invalid Azure token  | Return 401 Unauthorized error and `MissingRequestParam` message appears in log           | Integration               | <ul><li> [ ] Tested</li></ul>| Yes       |
```

Comment:
should be `missing "azure_token" in request body`

Reply:
Also the `when` is incorrect. Should be `VM sends authentication request to Conjur without an Azure token`

Reply:
@shaharglazner what do you think of this use-case? It looks more like a `400 Bad Request` to me as the problem is in the request itself.